components/PlaybackTask.brs

```diff
@@ -285,9 +285,24 @@ function playTrueXAd(adContainer as Object, truexAdInfo as Object, slotType = "p
     ' get channel's design resolution
     rect = m.top.getScene().currentDesignResolution
 
+    ' there are 2 ways how `adParameters` might be defined
+    ' - as `Creative[id="super_tag"].Linear.AdParameters` section in VAST - as json string
+    ' - as `Creative[id="super_tag"].CompanionAds.Companion.StaticResource[creativeType="application/json"]` as base64 encoded json string
+    if truexAdInfo.adparameters <> invalid then
+        adParameters = ParseJson(truexAdInfo.adparameters)
+    else
+        encodedAdParameters = truexAdInfo.companionads[0].url.Split("data:application/json;base64,")[1]
```

Comment:
should we be tolerant, handle if the companion ad is not present

i.e. if hasValidCompanion is not true, we should not be parsing things, that will cause a null pointer error

Reply:
@rblaa this function invoked only if isTruexAd(adInfo) passed - so, it's tollerant